olm-catalog/serverless-operator/project.yaml

```diff
@@ -21,6 +21,7 @@ requirements:
     - '4.6'
 
 dependencies:
+  knative-release: 0.17.0
```

Comment:
How did you determine 0.17.0? Just by looking at the CSV? It's confusing to have knative-release and also `eventing` and `serving` separately. Maybe there would be a better name for this variable.

Reply:
This is required in one of the names of our images. A bit weird, yes.

Reply:
As of Slack discussion: Removing this as this is used only in single, one time clean job specific to 0.17 release.